src/jaguar.toit

```diff
@@ -246,27 +280,6 @@ compute-timeout defines/Map --disabled/bool -> Duration?:
     logger.error "invalid $JAG-TIMEOUT setting ($jag-timeout)"
   return disabled ? (Duration --s=10) : null
 
-run-to-completion name/string? container/containers.Container start/int timeout/Duration?:
-  nick := name ? "container '$name'" : "program $container.id"
-
-  // We're only interested in handling the timeout errors, so we
-  // unwind and produce a stack trace in all other cases.
-  filter ::= : it != DEADLINE-EXCEEDED-ERROR
-
-  // Wait until the container is done or until we time out.
-  code/int? := null
-  catch --unwind=filter --trace=filter:
-    with-timeout timeout: code = container.wait
-  if not code:
-    elapsed ::= Duration --us=Time.monotonic-us - start
-    code = container.stop
-    logger.info "$nick timed out after $elapsed"
-
-  if code == 0:
-    logger.info "$nick stopped"
-  else:
-    logger.error "$nick stopped - exit code $code"
-
 run-code image-size/int reader/reader.Reader defines/Map --crc32/int -> none:
   jag-disabled := defines.get JAG-NETWORK-DISABLED
```

Comment:
Are you using `jag-disabled` and the `timeout` for anything? It smells like these 3 lines aren't needed anymore.